types/context.go

```diff
@@ -39,6 +39,8 @@ type Context struct {
 	consParams    *tmproto.ConsensusParams
 	eventManager  *EventManager
 	priority      int64 // The tx priority, only relevant in CheckTx
+	txCompletionChannels [][]chan interface{}
```

Comment:
how will we be using these to know when to block / signal on resource need / release

Reply:
Updated it to be a map of aclops -> []channels